templates/admin/team_details.html

```diff
@@ -92,11 +92,40 @@
   {% else %}
     <h3>{{year}}:</h3>
   {% endif %}
+  <ul>
   {% for media in medias %}
-    {% include "partials/admin_media_display.html" %}
+    <li>{% include "partials/admin_media_display.html" %}</li>
   {% endfor %}
-  <br>
+  </ul>
 {% endfor %}
 
+<div class="modal fade" id="confirm-delete" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                Confirm media reference removal
+            </div>
+            <div class="modal-body">
+                Are you sure you want to remove this reference to this media? References to other models will persist.
+            </div>
+            <div class="modal-footer">
+                <form id="delete_media_form" action="/admin/media/delete_reference" method="post">
+                <input name="reference_type" type="hidden" value="team" />
+                <input name="reference_key_name" type="hidden" value="{{team.key_name}}" />
+                <input name="originating_url" type="hidden" value="/admin/team/{{team.team_number}}" />
+
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                <button class="btn btn-danger" type="submit"><span class="glyphicon glyphicon-delete"></span> Remove reference</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+$('#confirm-delete').on('show.bs.modal', function(e) {
```

Comment:
This modal is cool. We should use it in more places when confirming actions (right now we redirect to separate pages and it's clunky).